v2/tools/generator/internal/codegen/pipeline/create_backwards_compatibility_types.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astmodel"
+)
+
+const CreateBackwardCompatibilityTypesStageID = "createStorageTypes"
```

Comment:
minor: Wrong stage ID?

Reply:
```suggestion
const CreateBackwardCompatibilityTypesStageID = "createBackwardCompabilityTypes"
```

Reply:
also minor: Wondering about the name of this. Would it be more grammatically correct to say `createBackwardsCompatibleStorageTypes` or something?

The "create backwards compatibility" for some reason feels like it should read "create backwards compatible" to me

Reply:
Updated naming for clarity.